xil-Town05/docker-compose.yml

```diff
@@ -42,32 +40,8 @@ services:
       - ROS_MASTER_URI=http://172.3.0.2:11311/
     command: roscore
 
-  platform_1:
```

Comment:
I thought that since `platform_1` was removed from the 'docker-compose.yml file', all ROS2 updates for the CARMA Platform were done. So, the idea was to use just the ROS1-ROS2 bridge to let the CARMA-CARLA integration tool talk directly to `platform_ros2`. But when I checked the [bridge.yml](https://github.com/usdot-fhwa-stol/carma-config/blob/tt-168-test-platform-with-spat/xil_carma_messenger/bridge.yml), I saw that not all ROS topics and services are being covered. It might be a good idea to look at the topics and services the CARMA-CARLA tool uses and make sure they match the 'bridge.yml' file before we remove `platform_1`. But this can be ignored if it has been tested already without any issue

Reply:
That is a good point, currently not seeing any issues with running the vehicle in sim with this config. However these topics/services would be good to add to the bridge.yml files. I will add them.